site/content/en/docs/enterprise/social-accounts-configuration.md

```diff
@@ -106,32 +106,61 @@ There are 2 basic steps to enable GitHub account authentication.
 > but don't forget to add required permissions.
 > <br>In the **Permission** > **Account permissions** > **Email addresses** must be set to **read-only**.
 
-## Enable authentication with an Amazon Cognito
-
-To enable authentication, do the following:
-
-1. Create a user pool. For more information,
-   see [Amazon Cognito user pools](https://docs.aws.amazon.com/cognito/latest/developerguide/cognito-user-identity-pools.html)
-2. Fill in the name field, set the homepage URL (for example: `https://localhost:8080`),
-   and authentication callback URL (for example: `https://localhost:8080/api/auth/social/amazon-cognito/login/callback/`).
-3. Create conпiguration file in CVAT:
-
-   1. Create the `auth_config.yml` file with the following content:
+## Enable Authentication with Amazon Cognito
+
+To enable authentication with Amazon Cognito for your CVAT instance, you need to complete two main steps:
+
+1. **Create and configure an [Amazon Cognito user pool](https://docs.aws.amazon.com/cognito/latest/developerguide/cognito-user-identity-pools.html)**.
+   Follow these detailed steps to set up the user pool:
+   - Create a user pool. For example, you might use settings like those shown in the image below:
```

Comment:
This seems to start in the middle. I would assume the first step would be "go to the AWS Management Console", then "click XXX to go to Cognito settings"...

Also, the steps should be in a numbered list, since they are sequential.